config/config.go

```diff
@@ -16,6 +16,8 @@ type Config struct {
 	PostgresURL             string        `envconfig:"POSTGRES_URL"`
 	SecretKey               string        `envconfig:"SECRET_KEY"`
 	ShutdownTimeout         time.Duration `envconfig:"SHUTDOWN_TIMEOUT"`
+	DatasetAPIURL           string        `envconfig:"DATASET_API_URL"`
```

Comment:
The README.md needs to be updated with these new configurations (line 19 and 20)

Reply:
done